README.md

```diff
@@ -86,37 +85,44 @@ the variables are named and described below:
 ### `consul_bin_path`
 
 - Binary installation path
-- Default value: `/usr/local/bin`
+- Default linux value: `/usr/local/bin`
+- Default windows value: `C:\ProgramData\consul\bin`
```

Comment:
I've used `C:\ProgramData\consul` for all necessary program data, including the consul.exe binary. Most Windows programs use the magical `%APPDATA%` path, but those are tied to specific users. My understanding is that `C:\ProgramData` is what Chocolatey and NSSM use, and it's a good place for system-level services.